src/app/lib/models/content_item.js

```diff
@@ -31,7 +31,7 @@
             var torrents = this.get('torrents');
 
             if (!torrents) {
-                console.error('tried to update health, but still no torrents here', this);
+                //console.error('tried to update health, but still no torrents here', this);
```

Comment:
this is actually a real issue that should be fixed... if we see this we're trying to render too soon